In [11]:
import pandas as pd
import requests
from bs4 import BeautifulSoup

In [12]:
url = 'https://www.amazon.in/s?k=mobile+under+20000&ref=nb_sb_noss_2'

In [14]:
page = requests.get(url)
page

<Response [503]>

### as i am getting response 503 so not scrapping data from site as per the instruction mentioned during webinar